### ISSUE TITLE
Doesn't work with webpack

### ISSUE THREAD

**Issue by bitplanets** [trentm/node-bunyan#213]
Module not found: Error: Cannot resolve module 'fs' in node_modules\bunyan\node_modules\mv\node_modules\mkdirp

The 2 modules that doesn't work are:

 - fs
 - dtrace-provider

https://github.com/trentm/node-bunyan/blob/2f95bb8091a8085756cc1f73c51dcf020f1990a2/lib/bunyan.js#L35

Look at: https://github.com/webpack/webpack/issues/689

**Comment by romanmt** [comment]
I also can't get client side bunyan working with wepack (IE10). For me the culprit is the ncp module which is required by the mv module. There's a closed issue on ncp so I believe the latest version doesn't have the offending line of code. May just need to update modules. 
https://github.com/AvianFlu/ncp/issues/56

**Comment by DenisIzmaylov** [comment]
Seems it steel doesn't work.
```javascript
{
  // ... package.json
  "dependencides": {
    "bunyan": "^1.4.0",
    "webpack": "^1.9.12",
    // ...
  }
}
```

**Comment by trentm** [comment]
Should be fixed with #282. Will be in bunyan v1.5.0.
